specification/client/models/common.models.tsp

```diff
@@ -124,3 +126,22 @@ union DotNetChatVoiceIds {
 union DotNetRealtimeVoiceIds {
   VoiceIdsShared,
 }
+
+// CollectionQueryParameters
+
+alias CollectionLimitQueryParameter = {
+  /**
+   * A limit on the number of objects to be returned. Limit can range between 1 and 100, and the
+   * default is 20.
+   */
+  @query pageSizeLimit?: int32 = 20;
+};
+
+alias CollectionAfterQueryParameter = {
+  /**
+   * A cursor for use in pagination. `after` is an object ID that defines your place in the list.
+   * For instance, if you make a list request and receive 100 objects, ending with obj_foo, your
+   * subsequent call can include after=obj_foo in order to fetch the next page of the list.
+   */
+  @query @continuationToken afterId?: string;
```

Comment:
Should this name match the REST API: `afterId` vs. `after` ?

Reply:
same as above